compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -4500,14 +4506,35 @@ private void handleInputClauseVariables(BLangInputClause bLangInputClause, Symbo
 
     private void visitCheckAndCheckPanicExpr(BLangCheckedExpr checkedExpr) {
         String operatorType = checkedExpr.getKind() == NodeKind.CHECK_EXPR ? "check" : "checkpanic";
```

Comment:
Can we extract `check` and `checkpanic` extracted to variables? It seems `check` has been used couple of places

Reply:
@nadeeshaan Did you mean to separate `check` and `checkpanic` variables into two separate variables?